dracut/modules.d/99kiwi-lib/module-setup.sh

```diff
@@ -15,7 +15,7 @@ depends() {
 install() {
     declare moddir=${moddir}
     inst_multiple \
-        blkid blockdev dd mkdir rmdir \
+        blkid partx blockdev dd mkdir rmdir \
```

Comment:
sorry @schaefi this change is not good yet, in kiwi-lib we are not replacing blockdev by partx completely. `blockdev` is still used to get block devices size. I could find a way to do the same with `partx`, hence we can't replace one by the other, we need both. In fact the current change as is in this commit has potential to break expandable disks. See where blockdev is used:

```bash
grep -rn blockdev dracut/modules.d/
dracut/modules.d/99kiwi-lib/module-setup.sh:18:        blkid blockdev dd mkdir rmdir \
dracut/modules.d/99kiwi-lib/kiwi-partitions-lib.sh:244:    echo $(($(blockdev --getsize64 "${device}") / 1024))
dracut/modules.d/99kiwi-lib/kiwi-partitions-lib.sh:253:    disk_bytes=$(blockdev --getsize64 "${disk}")
dracut/modules.d/99kiwi-lib/kiwi-partitions-lib.sh:284:            part_bytes=$((part_bytes + $(blockdev --getsize64 "${part}")))
```

`blockdev --getsize64` is a critical call. We can't risk missing blockdev in kiwi-lib. Probably some other module already pulls it in, hence adding it here probably does not make a difference in most cases, but still a hard requirement we need to include.

Reply:
I know and I did not remove it from kiwi-lib. Maybe you overlooked it but blockdev is still in the list to be included in module-setup.sh. blockdev got only deleted from the kiwi-live module as it is not used there anymore. Or did I miss something ?

Reply:
Damn it! you are absolutely right :+1: I don't know how I looked at diff that I had impression that partx was not added but subsituted. Sorry for the noise 🙏🏽